packages/ra-ui-materialui/src/auth/Login.tsx

```diff
@@ -114,18 +104,24 @@ class Login extends Component {
 
     componentDidUpdate() {
         if (!this.backgroundImageLoaded) {
-            this.lazyLoadBackgroundImage(true);
+            this.lazyLoadBackgroundImage();
         }
     }
 
     render() {
-        const { classes, className, loginForm, ...rest } = this.props;
+        const {
+            backgroundImage,
+            classes,
+            className,
+            loginForm,
+            ...rest
+        } = this.props;
 
         return (
             <MuiThemeProvider theme={this.theme}>
                 <div
                     className={classnames(classes.main, className)}
-                    {...sanitizeRestProps(rest)}
+                    {...rest}
```

Comment:
why did you remove the sanitization?

Reply:
That means that the type of the Props for that component is too restrictive, and will warn the user when they pass other props. You should extend the prop types of the child to whom you pass the rest props.